src/main.ts

```diff
@@ -77,6 +77,7 @@ class LocalTableland {
         // we can't run in windows if we use detached mode
         detached: !isWindows(),
         cwd: this.registryDir,
+        env: { ...process.env, HARDHAT_NETWORK: "hardhat" },
```

Comment:
Long story, but in the case the host code using local tableland as a library calls this code to start local tableland, the parent process env is inherited. If that env happens to have a hardhat network defined that isn't `hardhat`, this will fail to start the local hardhat node. So I'm overriding it here to make it explicitly `hardhat` no matter what the parent process provides.